### PR TITLE
Fixes #6337 - VM tab is hidden when we deselect profile

### PR DESCRIPTION
**Pull request by dLobatog**
The event `computeResourceSelected` is called `onChange` for both the Compute Resource selector and the Compute Profile selector in the Host#edit form. Before this patch, choosing a Compute Profile, then no Compute Profile (empty entry in the dropdown) would close the Virtual Machine tab, even though a Compute Resource was still selected. 

It required a minor tweak to make the function only close the Virtual Machine tab only when the Compute Resource selector goes to Bare Metal. When Compute Profile "None" is selected, it just removes all settings set by any other Compute Profile previously.

### CODE REVIEW

File: app/views/hosts/_form.html.erb [hunk 1]
```diff
@@ -64,7 +64,7 @@
          be different from the defaults, or the defaults could have changed after the vm was created. %>
         <div id="compute_profile" <%= display?(!@host.compute_resource_id) %> >
           <%= select_f f, :compute_profile_id, ComputeProfile.visibles, :id, :name,
-            { :include_blank => true },
+            { :include_blank => 'None' },
```
Comment (domcleal) [inline app/views/hosts/_form.html.erb]: missing `_()`

Comment (domcleal) [inline app/views/hosts/_form.html.erb]: Is this change required if the JS change is present?  Just wondering if we can leave it out for consistency with the rest of the UI, which uses true/a blank entry.

**Comment by dLobatog** [comment]
@domcleal "None" wasn't required, I just thought it looked better, I removed it now.

**Comment by domcleal** [comment]
Thanks @eLobato, merged as c7a7004 for Foreman 1.5.2.
